apps/package.json

```diff
@@ -222,11 +222,14 @@
     "@dnd-kit/sortable": "^10.0.0",
     "@dnd-kit/utilities": "^3.2.2",
     "@dsco_/link": "^1.1.2",
+    "@emotion/react": "^11.14.0",
```

Comment:
Do we need to add these to package.json or can we just rely on the packages to get added by dependency on MUI?